src/wasm-v8.cc

```diff
@@ -1200,6 +1200,11 @@ auto Ref::copy() const -> own<Ref*> {
   return impl(this)->copy();
 }
 
+auto Ref::same(const Ref* that) const -> bool {
+  v8::HandleScope handle_scope(impl(this)->isolate());
+  return impl(this)->v8_object()->SameValue(impl(that)->v8_object());
```

Comment:
Why not `impl(this)->v8_object() == impl(that)->v8_object()`? IIUC, `SameValue` boils down to pointer equality for all subtypes of `Ref` anyway, and using `Local::operator==` directly would make that more obvious.

Reply:
If you actually use this in conjunction with JS, then arbitrary JS values can arise as host references, and then the expected semantics for reference identity is that of JS same-value.